### ISSUE TITLE
[Accessibility] Audio Cues still doesn't work in github.dev insider

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#11184]
CC -isidorn

Following -147190

I think the latest GitHub Dev insiders has already picked up the audio cues web patch, given that I can see the latest audio cues in the list, such as "Task Completed" and "Task Failed"

![image](https://user-images.githubusercontent.com/19754711/199735912-74caa892-b634-4553-80ef-a2fb6c190d61.png)

-- sysem Info

Version: 1.74.0-insider
Commit: 86f0f9bfa268fe21af7efae37d8f0da1dba8137f
Date: 2022-11-03T05:23:43.231Z
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36
-- Reproducible Steps

1. Go to https://github.dev/microsoft/vscode

1. Launch the GitHub Dev with insider mode.

1. Launch "Help: List Audio Cues"

* Notes: audio cues are not playing.
